src/query.ts

```diff
@@ -86,8 +87,19 @@ export class Query {
    * @param  {String}   query   The search query to be sent.
    *
    */
-  public search(query: string): void {
+  public search(query: string): Query {
```

Comment:
Use `this` as the return type. The same for all the other chainable methods. Also, other methods that return `void` probably should be chainable too…